test/Models/SdmxJsonCatalogItemSpec.js

```diff
@@ -238,13 +241,15 @@ describe('SdmxJsonCatalogItem', function() {
                 // Expect it to have realised this has NO regional data.
                 var regionDetails = item.regionMapping.regionDetails;
                 expect(regionDetails).not.toBeDefined();
-                expect(item.isMappable).toBe(false);
-                // Expect it to have created the right table of data (with no time dimension).
-                var columnNames = item.tableStructure.getColumnNames();
-                expect(columnNames.length).toEqual(2 + 5 * 2 * 2); // one for each value, plus date and total columns.
-                expect(item.tableStructure.columns[0].values.length).toEqual(3); // 3 dates.
-                // Expect it to show 3 concepts to the user.
-                expect(item.concepts.length).toEqual(3);
+                // Beyond this, assume nothing. But if you want it to work, you may want the following:
```

Comment:
Non-spatial SDMXs passed this test but gave weird results before. So I made them _really_ not work instead, so the rest of this test fails, but the user gets a more sensible experience.